crawl-ref/source/makeitem.cc

```diff
@@ -647,6 +643,7 @@ bool is_missile_brand_ok(int type, int brand, bool strict)
     case SPMSL_EXPLODING:
         return type == MI_TOMAHAWK;
     case SPMSL_STEEL: // deliberate fall through
+		return false;
```

Comment:
I think this might be what checkwhite is finding (see the travis failure), extra whitespace here. In any case, you can run the `util/checkwhite` script to fix whitespace errors

Reply:
(Specifically the issue here is using tabs instead of spaces.)

Reply:
I'll correct it and try again. Thank you for your advice.